addons/l10n_in_edi/views/edi_pdf_report.xml

```diff
@@ -1,31 +1,29 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <template id="l10n_in_einvoice_external_layout_standard_inherit" inherit_id="web.external_layout_standard">
+        <div name="company_address" position="after">
+            <div class="col-6" t-if="l10n_in_einvoice_json">
+                <div class="d-flex justify-content-center">
+                    <div class="text-center">
+                        <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
+                        ('QR', l10n_in_einvoice_json['SignedQRCode'], 200, 200)"/><br/>
+                        <span t-esc="l10n_in_einvoice_json['Irn']"/>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </template>
     <template id="l10n_in_einvoice_report_invoice_document_inherit" inherit_id="account.report_invoice_document">
-        <xpath expr="//div[@id='informations']" position="before">
+        <xpath expr="//div[@id='informations']" position="inside">
             <t t-set="l10n_in_einvoice_json" t-value="o._get_l10n_in_edi_response_json()"/>
-            <div id="l10n_in_einvoice_informations_inr" class="row mt-4 mb-4">
-                <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="irn">
-                    <strong>IRN:</strong>
-                    <p class="m-0" t-esc="l10n_in_einvoice_json['Irn']"/>
-                </div>
+            <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_no">
+                <strong>Ack. No:</strong>
+                <p class="m-0" t-esc="l10n_in_einvoice_json['AckNo']"/>
             </div>
-            <div id="l10n_in_einvoice_informations_other" class="row mt-4 mb-4">
-                <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_no">
-                    <strong>Ack. No:</strong>
-                    <p class="m-0" t-esc="l10n_in_einvoice_json['AckNo']"/>
-                </div>
-                <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_date">
-                    <strong>Ack. Date:</strong>
-                    <p class="m-0" t-esc="l10n_in_einvoice_json['AckDt']"/>
-                </div>
+            <div class="col-auto col-3 mw-100 mb-2" t-if="l10n_in_einvoice_json" name="ack_date">
+                <strong>Ack. Date:</strong>
+                <p class="m-0" t-esc="l10n_in_einvoice_json['AckDt']"/>
             </div>
         </xpath>
-        <xpath expr="//div[@id='qrcode']" position="after">
```

Comment:
why did not we put it inside?

Reply:
normally there is a payment QR code so it's confusion for end user